core/src/test/java/org/opensearch/sql/data/model/DateTimeValueTest.java

```diff
@@ -136,41 +125,11 @@ public void timestampInUnsupportedFormat() {
         exception.getMessage());
   }
 
-  @Test
-  public void datetimeInUnsupportedFormat() {
-    SemanticCheckException exception =
-        assertThrows(
-            SemanticCheckException.class, () -> new ExprDatetimeValue("2020-07-07T01:01:01Z"));
-    assertEquals(
-        "datetime:2020-07-07T01:01:01Z in unsupported format, "
-            + "please use 'yyyy-MM-dd HH:mm:ss[.SSSSSSSSS]'",
-        exception.getMessage());
-  }
-
-  @Test
-  public void stringDateTimeValue() {
-    ExprValue stringValue = new ExprStringValue("2020-08-17 19:44:00");
-
-    assertEquals(LocalDateTime.parse("2020-08-17T19:44:00"), stringValue.datetimeValue());
-    assertEquals(LocalDate.parse("2020-08-17"), stringValue.dateValue());
-    assertEquals(LocalTime.parse("19:44:00"), stringValue.timeValue());
-    assertEquals("\"2020-08-17 19:44:00\"", stringValue.toString());
-
-    SemanticCheckException exception =
-        assertThrows(
-            SemanticCheckException.class,
-            () -> new ExprStringValue("2020-07-07T01:01:01Z").datetimeValue());
-    assertEquals(
-        "datetime:2020-07-07T01:01:01Z in unsupported format, "
-            + "please use 'yyyy-MM-dd HH:mm:ss[.SSSSSSSSS]'",
-        exception.getMessage());
-  }
-
   @Test
   public void stringDateValue() {
     ExprValue stringValue = new ExprStringValue("2020-08-17");
 
-    assertEquals(LocalDateTime.parse("2020-08-17T00:00:00"), stringValue.datetimeValue());
+    assertEquals(LocalDate.parse("2020-08-17"), stringValue.dateValue());
```

Comment:
That test was actually change to date rather than timestamp, I have updated the test in https://github.com/opensearch-project/sql/pull/1980/commits/20c084226b9018820e6eba09a003ba407166f2f4